homeassistant/components/anova/__init__.py

```diff
@@ -36,36 +35,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
         return False
     assert api.jwt
-    api.existing_devices = [
-        AnovaPrecisionCooker(
-            aiohttp_client.async_get_clientsession(hass),
-            device[0],
-            device[1],
-            api.jwt,
-        )
-        for device in entry.data[CONF_DEVICES]
-    ]
     try:
-        new_devices = await api.get_devices()
+        await api.create_websocket()
     except NoDevicesFound:
-        # get_devices raises an exception if no devices are online
-        new_devices = []
-    devices = api.existing_devices
-    if new_devices:
-        hass.config_entries.async_update_entry(
-            entry,
-            data={
-                **entry.data,
-                CONF_DEVICES: serialize_device_list(devices),
-            },
+        _LOGGER.warning(
+            "No devices were found on the websocket, perhaps you don't have any devices on this account?"
         )
+    # Create a coordinator per device, if the device is offline, no data will be on the websocket,
+    # and the coordinator should auto mark as unavailable. But as long as the websocket successfully connected,
+    # config entry should setup.
```

Comment:
Let's respect our preferred line length limit
```suggestion
    # Create a coordinator per device, if the device is offline, no data will be on the
    # websocket, and the coordinator should auto mark as unavailable. But as long as the
    # websocket successfully connected, config entry should setup.
```